clambcc/clambc-compiler.py

```diff
@@ -260,8 +277,15 @@ def linkIRFiles(clangLLVM: ClangLLVM, linkedFile: str, irFiles: list) -> int:
     Given an output file name and list of IR files, link the IR files.
     Returns the exit status code for the call to `llvm-link`.
     '''
-    inFiles = " ".join(irFiles)
-    cmd = f"{clangLLVM.getLLVMLink()} -S -o {linkedFile} {inFiles}"
+    cmd = []
+    cmd.append(clangLLVM.getLLVMLink())
+    cmd.append("-S")
+    cmd.append("-o")
+    cmd.append(linkedFile)
+    cmd += irFiles
+
+    #TODO: Remove (FUTURE VERSION)
```

Comment:
Why remove this in a future version?  Can you elaborate in the comment so a future engineer will understand? The commit message itself won't have enough context to explain.

Reply:
udpated